alpine/entrypoint.sh

```diff
@@ -12,4 +12,4 @@ if traefik "$1" --help 2>&1 >/dev/null | grep "help requested" > /dev/null 2>&1;
     set -- traefik "$@"
 fi
 
-exec "$@"
\ No newline at end of file
+exec $(echo $(eval echo $@))
```

Comment:
Passing a shellcheck to the proposed entrypoint.sh script raised an errors and some warnings:

* The warnings are about double-quoting the results of sub commands (the `$(...)` instructions).

* The error which catch my attention is https://github.com/koalaman/shellcheck/wiki/SC2005 .

Fixing all leads into this instruction: `exec "$(eval echo "$@")"` : is it still working for you?